stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -256,7 +256,7 @@ public struct _ContiguousArrayBuffer<Element> : _ArrayBufferProtocol {
 
   /// If the elements are stored contiguously, a pointer to the first
   /// element. Otherwise, `nil`.
-  public var firstElementAddress: UnsafeMutablePointer<Element> {
+  public var firstElementAddress: UnsafeMutablePointer<Element>? {
```

Comment:
I think it falls under <rdar://problem/17906373> [Protocol conformance] Allow a witness whose type is a subtype of the corresponding requirement's type.